driver/controller.go

```diff
@@ -242,12 +242,15 @@ func (s controllerService) GetCapacity(ctx context.Context, req *csi.GetCapacity
 		var err error
 		capacity, err = s.nodeService.GetTotalCapacity(ctx, deviceClass)
 		if err != nil {
-			return nil, status.Error(codes.Internal, err.Error())
+			ctrlLogger.Error(err, "get total capacity failed")
+			return &csi.GetCapacityResponse{AvailableCapacity: 0}, nil
```

Comment:
https://kubernetes.slack.com/archives/C09QZFCE5/p1622273728215500
> My expectation was that a CSI driver returns "no capacity" = 0 when it cannot create volumes with the requested parameters.
> As you said, there is no specific error code for this scenario whereas 0 is a valid and correct answer.

Reply:
I understand it is expected not an error. Nice clarification.

Reply:
GetTotalCapacity returns a real error if getNodes failed.
https://github.com/topolvm/topolvm/blob/c974ca0f99e497a83b450ccdae369d0262962f48/driver/k8s/node_service.go#L80
And also, if deviceClass annotation is missing, this code returns 0, so this change is not needed, IMO.
https://github.com/topolvm/topolvm/blob/c974ca0f99e497a83b450ccdae369d0262962f48/driver/k8s/node_service.go#L85-L90